source/workflowapi/app.py

```diff
@@ -2752,6 +2752,170 @@ def create_terminology():
     response['TerminologyProperties']['LastUpdatedAt'] = response['TerminologyProperties']['LastUpdatedAt'].isoformat()
     return response
 
+# ===== ACT
+@app.route('/service/translate/get_parallel_data', cors=True, methods=['POST'], content_types=['application/json'], authorizer=authorizer)
+def get_parallel_data():
+    """ Get a link to the CSV formatted description for an Amazon Translate Parallel Data Set.
+
+    Body:
+
+    .. code-block:: python
+
+        {
+            'parallel_data_name'='string'
+        }
+
+    Returns:
+        This is a proxy for boto3 get_parallel_data and returns the output from that SDK method.  
+        See `the boto3 documentation for details <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/translate.html#Translate.Client.get_parallel_data>`_
+
+    Raises:
+        See the boto3 documentation for details 
+        500: ChaliceViewError - internal server error
+    """
+    print('get_parallel_data request: '+app.current_request.raw_body.decode())
+    translate_client = boto3.client('translate', region_name=os.environ['AWS_REGION'])
+    parallel_data_name = json.loads(app.current_request.raw_body.decode())['parallel_data_name']
+    response = translate_client.get_parallel_data(Name=parallel_data_name, parallel_dataDataFormat='CSV')
+    # Remove response metadata since we don't need it
+    del response['ResponseMetadata']
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Your code is attempting to access the response metadata, `ResponseMetadata`, without performing a `None` check on the response object. This might cause a NoneType TypeError. To prevent this, we recommend that you add a `None` check on the response object before accessing the response metadata.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html)